src/block_mixins/uploadable.js

```diff
@@ -19,6 +19,15 @@ module.exports = {
 
     this.upload_options = Object.assign({}, config.defaults.Block.upload_options, this.upload_options);
     this.inputs.insertAdjacentHTML("beforeend", _.template(this.upload_options.html, this));
+
+    Array.prototype.forEach.call(this.inputs.querySelectorAll('button'), function(button) {
```

Comment:
Probably best not, as ST doesn't have a shim for it already and we don't want to add one ideally?